### PR TITLE
add combinations and permutations

### PR DESCRIPTION
**Pull request by mayurdahibhate**
Added ```combinations``` and ```permutations``` algorithms to the ```math``` folder. Added test according ```CONTRIBUTING.md```.

### CODE REVIEW

**Review by arubhardwaj — DISMISSED** [review]

File: src/math/combination.jl [hunk 1]
```diff
@@ -0,0 +1,31 @@
+"""
+    combination(n, r)
+        
+returns the number of ways picking r unordered outcomes from
+n possibilities, without repetition
+
+# Arguments:
+- `n`: Positive integers of items to choose from
+- 'r': Positive integers of items to choose
```
Comment (Panquesito7) [inline src/math/combination.jl:9]:
```suggestion
- 'r': Positive integers of items to choose

Contributed By:- [Mayur Dahibhate](https://github.com/mayurdahibhate)
```

File: src/math/permutation.jl [hunk 2]
```diff
@@ -0,0 +1,31 @@
+"""
+    permutation(n, r)
+        
+    returns the number of ways to choose r items from n items
+    with order and without repetition
+
+# Arguments:
+- `n`: Positive integers of items to choose from
+- 'r': Positive integers of items to choose
```
Comment (Panquesito7) [inline src/math/permutation.jl:9]:
```suggestion
- 'r': Positive integers of items to choose

Contributed By:- [Mayur Dahibhate](https://github.com/mayurdahibhate)
```

**Review by Panquesito7 — DISMISSED** [review]
Looks good! Thank you for your contribution! 👍
@Azzaare, if you have time, please review this PR. Thanks. 🙂

**Comment by Azzaare** [comment]
@mayurdahibhate Thank you for the PR! It looks fine beside one key point... the new files for combination and permutation are not included in the main source file `src/TheAlgorithms.jl`

Could you update your PR :)

**Review by arubhardwaj — COMMENTED** [review]
It looks good, once your make the changes suggested by @Azzaare I will merge it

**Comment by mayurdahibhate** [comment]
> It looks good, once your make the changes suggested by @Azzaare I will merge it

I have already included new files in the main source files as suggested by @Azzaare.

**Review by arubhardwaj — APPROVED** [review]
awesome then

**Pull request merged**